deploy/sdk/src/dynamo/sdk/core/protocol/deployment.py

```
@@ -137,13 +139,28 @@ class Deployment:
     pipeline: t.Optional[str] = None
     entry_service: t.Optional[Service] = None
-    envs: t.Optional[t.List[dict]] = None
+    envs: t.Optional[t.List[t.Dict[str, t.Any]]] = None
 
 
 # Type alias for deployment responses (e.g., from backend APIs)
 DeploymentResponse = t.Dict[str, t.Any]
 
 
+@dataclass
+class DeploymentConfig:
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)